packages/cactus-plugin-ledger-connector-quorum/src/main/typescript/plugin-ledger-connector-quorum.ts

```diff
@@ -12,6 +12,10 @@ import Web3JsQuorum, {
   ISendRawTransaction,
   IPrivateTransactionReceipt,
 } from "web3js-quorum";
+import {
+  WebsocketProviderOptions,
+  HttpProviderOptions,
+} from "web3-core-helpers";
```

Comment:
@rwat17 Please declare this dependency in the package.json of the connector as well. While working on the dev build it is more or less guaranteed that this package will be available in the project root's `node_modules` but there are other situations in which it might not be present unless we explicitly specify the dependency.